detect_test.go

```diff
@@ -97,6 +97,11 @@ func TestDetectAttachmentHeader(t *testing.T) {
 			header: textproto.MIMEHeader{
 				"Content-Disposition": []string{"inline"}},
 		},
+		{
+			want: false,
+			header: textproto.MIMEHeader{
+				"Content-Disposition": []string{"inline; broken"}},
+		},
```

Comment:
Add another entry to test `"attachment; broken"`

Reply:
Done.